[ONNX] Remove unnecessary deepcopy on args in 'DynamoExport'

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #104736

The comment is outdated. There should be no side-effects on `args` and `kwargs`.

BowenBao committed:
[ONNX] Remove unnecessary deepcopy on args in 'DynamoExport'

[ghstack-poisoned]

thiagocrepaldi commented:
@pytorchbot merge

thiagocrepaldi commented:
@BowenBao there is an unused `import copy` at `torch/onnx/_internal/fx/dynamo_graph_extractor.py`

BowenBao committed:
Update on "[ONNX] Remove unnecessary deepcopy on args in 'DynamoExport'"


The comment is outdated. There should be no side-effects on `args` and `kwargs`.

[ghstack-poisoned]

thiagocrepaldi commented:
@pytorchbot merge
